halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -71,6 +72,20 @@ public void setColor(boolean color) {
 
   private static JobExecutor jobExecutor;
 
+  private static String[] failureMessages = {
+    "I'm sorry " + System.getProperty("user.name") + ", I'm afraid I can't do that.",
+    "If at first you don't succeed, try again with a different set of flags.",
+    "Looks like you're trying to deploy Spinnaker. Would you like some help?"
+  };
+
+  private static void showRandomFailureMessage() {
+    if (ThreadLocalRandom.current().nextInt(0, 100) < 5) {
+      int index = ThreadLocalRandom.current().nextInt(0, failureMessages.length);
```

Comment:
given that the API directly supports setting bounds on the random int this seem neaters as is